src/launcher/index.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import {
+  Token
+} from 'phosphor/lib/core/token';
+
+import {
+  JSONObject
+} from 'phosphor/lib/algorithm/json';
+
+import {
+  DisposableDelegate, IDisposable
+} from 'phosphor/lib/core/disposable';
+
+/* tslint:disable */
+/**
+ * The main menu token.
+ */
+export const ILauncher = new Token<ILauncher>('jupyter.services.launcher');
+/* tslint:enable */
+
+/**
+ * The launcher interface.
+ */
+export interface ILauncher {
+  /**
+   * Add a command item to the Launcher
+   *
+   * @param name - The display name
```

Comment:
There should be a blank line between the param comments and a period at the end of each comment.